crates/aptos-logger/src/aptos_logger.rs

```diff
@@ -512,13 +534,13 @@ pub trait Writer: Send + Sync {
     fn write(&self, log: String);
 }
 
-/// A struct for writing logs to stderr
-struct StderrWriter;
+/// A struct for writing logs to stdout
+struct StdoutWriter;
 
-impl Writer for StderrWriter {
-    /// Write log to stderr
+impl Writer for StdoutWriter {
```

Comment:
writing to stdout instead since this is better practice

Reply:
Currently we write to a file for smoke tests and standalone test envs, right?

Reply:
no idea. I don't think we write to a file for any of the internal testnets though. For smoke we may.
Either way I don't think my changes interfere with that since there is a separate `FileWriter` struct which I haven't touched.

Reply:
the file based logging was btw introduced by you about 2 years ago https://github.com/aptos-labs/aptos-core/commit/7b30e4a59acb67fba8e6fe70b93ed2c0bd622c7c#diff-23673100bf3fcdf710542727e05d552bd10d2e1709a311ca33b354e22431afd1R123 :-P